dali/kernels/signal/window/extract_windows_gpu_test.cu

```diff
@@ -34,7 +34,7 @@ TEST(ExtractWindowsGPU, NonBatchedKernel) {
   int step = 10;
   int length = windows * step - 100;;
   int center = 5;
-  bool reflect = false;
+  const bool reflect = false;
```

Comment:
Actually, this makes me wonder why this is fixed at `false` - I'd give it a short at `true` and if it happens to work, have a loop:
```suggestion
  for (bool reflect : { false, true }) {
```

Reply:
Done